compiler/rustc_codegen_gcc/src/intrinsic/archs.rs

```diff
@@ -7210,29 +7210,6 @@ match name {
     "llvm.ve.vl.xorm.MMM" => "__builtin_ve_vl_xorm_MMM",
     "llvm.ve.vl.xorm.mmm" => "__builtin_ve_vl_xorm_mmm",
     // x86
-    "llvm.x86.3dnow.pavgusb" => "__builtin_ia32_pavgusb",
```

Comment:
This file is auto-generated, so it doesn't make sense to remove these.

Reply:
Shucks, just noticed it's in the GCC backend too. I'll undo that.